docs/vsphere_ci/build.json

```diff
@@ -50,6 +50,7 @@
         "scripts/install-docker.ps1",
         "scripts/install-firewall-rules.ps1"
       ],
+      "vm_version": 15,
```

Comment:
This should go in a separate PR. See [WINC-726](https://issues.redhat.com/browse/WINC-726)

Reply:
Opened https://github.com/openshift/windows-machine-config-operator/pull/895